proto/v1/response.proto

```
@@ -5,13 +5,17 @@ package valkey_timeseries.fanout.v1;
 import "v1/common.proto";
 
 message MGetValue {
-  string key = 1;
+  // Key names are binary-safe in Valkey (an interior NUL or a non-UTF-8 byte is
+  // legal), so this is `bytes`, not `string`: proto3 `string` requires valid
+  // UTF-8 and would force a lossy conversion that corrupts the name the client
+  // sees. Same for every other key-name field below.
+  bytes key = 1;
```

Buf (1.72.0)

[error] Field "1" with name "key" on message "MGetValue" changed type from "string" to "bytes". See https://developers.google.com/protocol-buffers/docs/proto3#updating for wire compatibility rules and https://developers.google.com/protocol-buffers/docs/proto3#json for JSON compatibility rules.

(FIELD_WIRE_JSON_COMPATIBLE_TYPE)